steampy/utils.py

```diff
@@ -72,6 +84,63 @@ def merge_items(items: List[dict], descriptions: dict, **kwargs) -> dict:
     return merged_items
 
 
+def get_market_listings_from_html(html: str) -> dict:
+    document = BeautifulSoup(html, "html.parser")
+    nodes = document.select("div[id=myListings]")[0].findAll("div", {"class": "market_home_listing_table"})
+    sell_listings_dict = {}
+    buy_orders_dict = {}
+    for node in nodes:
+        if "My sell listings" in node.text:
+            sell_listings_dict = get_sell_listings_from_node(node)
+        elif "My listings awaiting confirmation" in node.text:
+            sell_listings_awaiting_conf = get_sell_listings_from_node(node)
+            for listing in sell_listings_awaiting_conf.values():
+                listing["need_confirmation"] = True
+            sell_listings_dict.update(sell_listings_awaiting_conf)
+        elif "My buy orders" in node.text:
+            buy_orders_dict = get_buy_orders_from_node(node)
+    return {"buy_orders": buy_orders_dict, "sell_listings": sell_listings_dict}
+
```

Comment:
The nodes are these things marked in yellow [here](http://i.imgur.com/XHfk7fz.png) (I'm talking about the html obviusly)